modules/network/src/test/java/org/apache/ignite/internal/network/netty/RecoveryHandshakeTest.java

```diff
@@ -801,7 +815,19 @@ private RecoveryAcceptorHandshakeManager createRecoveryAcceptorHandshakeManager(
                 new ClusterNodeImpl(launchId, consistentId, new NetworkAddress(ACCEPTOR_HOST, PORT)),
                 MESSAGE_FACTORY,
                 provider,
-                () -> List.of(acceptorSideChannel.eventLoop()),
+                new HandshakeEventLoopSwitcher(List.of()) {
```

Comment:
Should we introduce a `NoOpHandshakeEventLoopSwitcher` in tests to avoid repetition?

Reply:
Done.